charts/Makefile

```diff
@@ -15,7 +15,7 @@ cert-manager:
 	kubectl create namespace cert-manager
 	helm repo add jetstack https://charts.jetstack.io
 	helm repo update
-	helm install cert-manager jetstack/cert-manager --version v1.1.0 \
+	helm install cert-manager jetstack/cert-manager --version v1.2.0 \
 		--set installCRDs=true \
 		--namespace cert-manager \
 		--wait --timeout 120s
```

Comment:
when we create a new kind cluster and there is no cert-manager namespace installed, maybe the following steps will be helpful to mention in the documentation: 
(1) kubectl create namespace cert-manager
(2) helm repo add jetstack https://charts.jetstack.io
(3) helm repo update

Reply:
Thanks. I updated the relevant section.